scripts/issue/1450/divest_aura.py

```diff
@@ -0,0 +1,61 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+# flag
+prod = True
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_cow(prod)
+
+    # tokens
+    weth = vault.contract(r.treasury_tokens.WETH)
+    gravi = vault.contract(r.sett_vaults.graviAURA)
+    aura = vault.contract(r.treasury_tokens.AURA)
+
+    # wd from gravi
+    gravi.withdrawAll()
+
+    aura_balance = aura.balanceOf(vault)
+
+    # build ladder orders
+    leg_amount = aura_balance // 3
+
+    # 7 days
+    deadline = 60 * 60 * 24 * 7
+
+    # approve whole aura stack
+    vault.cow.allow_relayer(aura, aura_balance)
+
+    # at market
+    vault.cow.market_sell(
```

Comment:
it might more concise using a loop with `coef` array since all other parameters for market_sell are the same